src/haz3lweb/view/LangDoc.re

```diff
@@ -672,20 +672,78 @@ let get_doc =
           ),
           [],
         );
-      | TypFun(_, _) =>
-        // TODO (typfun)
-        let (doc, options) =
-          LangDocMessages.get_form_and_options(
-            LangDocMessages.triv_exp_group,
-            docs,
-          );
-        get_message(
-          doc,
-          options,
-          LangDocMessages.triv_exp_group,
-          doc.explanation.message,
-          [],
-        );
+      | TypFun(_, _ /* tpat, body */) =>
+        basic_info(LangDocMessages.typ_function_group)
+      /* TODO: Cannot quite get the below to work. The EmptyHole case throws an unhandled exception Not_found.
```

Comment:
Might need some help here from someone that understands the LangDoc infrastructure better than me?